package.json

```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "axios": "^0.19.0",
     "bcryptjs": "^2.4.3",
+    "chart.js": "^2.9.3",
```

Comment:
Wondering if this can be removed? It doesn't seem to be used

Reply:
the readme for react-chart-js2 said that in recent updates they made chart.js a peer dependency, that it should be added to the dependencies. I didn't see it in the code you sent me though, so I wasn't sure.

Reply:
Gotcha!